src/Algebra/Solver/IdempotentCommutativeMonoid.agda

```diff
@@ -26,7 +26,7 @@ import Relation.Nullary.Decidable            as Dec
 import Data.Vec.Relation.Binary.Pointwise.Inductive as Pointwise
 
 open import Relation.Binary.PropositionalEquality as ≡ using (_≡_; decSetoid)
-open import Relation.Nullary.Decidable using (Dec)
+open Dec using (Dec; decToMaybe)
```

Comment:
For consistency with `Algebra.Solver.Monoid`, which uses `Dec.decToMaybe` qualified, maybe stick with the old `import` but with the qualified name `Dec`? or else expose `decToMaybe` at top-level in `...Monoid`?